src/main/scala/org/globalforestwatch/summarystats/carbonflux/CarbonFluxCommand.scala

```diff
@@ -20,7 +20,7 @@ object CarbonFluxCommand extends SummaryCommand {
         "outputUrl" -> default.outputUrl,
         "noOutputPathSuffix" -> default.noOutputPathSuffix,
         "changeOnly" -> changeOnly,
-        "config" -> GfwConfig.get
+        "config" -> GfwConfig.get(None)
```

Comment:
If you look in SummaryCommand, there's already a `pinnedVersions` input parameter to the geotrellis command. Should that be propagated instead? 

Also feels a little strange to have to pass None to each individual get function, I forget, does Scala not have default parameters?

Reply:
> Also feels a little strange to have to pass None to each individual get function, I forget, does Scala not have default > parameters?

Good suggestion about adding a default parameter for GfwConfig.get().  I've made that change.  Since GfwConfig.get does have an argument now, you do have use `GfwConfig.get()` everywhere now, rather than just `GfwConfig.get`.

> If you look in SummaryCommand, there's already a pinnedVersions input parameter to the geotrellis command.
> Should that be propagated instead?

In the new GfwProDashboardAnalysisSpec test, I specify a specific pinned version for the test, so of course, I don't need or use pinnedVersionsOpts (and the tests don't have access to it anyway)

I changed the code in GfwProDashboardCommand() to use pinnedVersionOpts as an example.  I doubt that any submitted analysis command will want to use pinned versions (other than for tests), so I didn't change all the other analyses, but let me know if you want me to do that.  Pinning only applies to the data sets that use `latest` as their version, which currently are only glad alerts and integrated alerts.